src/tasks/__init__.py

```diff
@@ -14,10 +14,15 @@
 from .tasks import Task
 
 ##
+# All cola-like tasks for which we need write_preds() and append to ALL_GLUE_TASKS below
+ALL_COLA_NPI_TASKS = ['cola_npi_x']
```

Comment:
If there's only one item, you don't need a list—just refer to this directly later.

Reply:
There should be nine. Still waiting for the data generation team's decision